docs/ide/dotfuscator/index.md

```diff
@@ -55,27 +57,27 @@ Examples of [.NET Obfuscation][obfuscation] and other [Application Protection][a
 
 For details on these features, including how they fit into your application protection strategy, see the [Capabilities page][capabilities].
 
-Dotfuscator CE offers basic protection out-of-the-box.
-Even more application protection measures are available to registered users of Dotfuscator CE,
-and to users of *PreEmptive Protection - Dotfuscator* Professional Edition, the world's leading [.NET Obfuscator][net-obfuscator].
+Dotfuscator Community offers basic protection out-of-the-box.
+Even more application protection measures are available to registered users of Dotfuscator Community,
+and to users of ***PreEmptive Protection - Dotfuscator Professional***, the world's leading [.NET Obfuscator][net-obfuscator].
 For information about enhancing Dotfuscator, see the [Upgrades page][upgrades].
 
 ## Getting started
 
-To begin using Dotfuscator CE from Visual Studio, type `dotfuscator` into the **Quick Launch** (Ctrl+Q) search bar.
+To begin using Dotfuscator Community from Visual Studio, type `dotfuscator` into the **Quick Launch** (Ctrl+Q) search bar.
```

Comment:
The search box in VS 2019 is now just the search box (it's not called Quick Launch anymore).